File: Practice/LeetCode/Easy/TwoSum.java
public class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer,Integer> map = new HashMap<Integer,Integer>();
        List<Integer> result = new ArrayList<Integer>();
        for(int i=0;i<nums.length;i++){
            if(map.containsKey(nums[i])){
                result.add(map.get(nums[i]));
                result.add(i);
                break;
            }
            else
            map.put(target-nums[i],i);
        }

        int[] res = new int[result.size()];
        for(int i=0;i<res.length;i++)
            res[i] = result.get(i);

            return res;
    }
}

File: Practice/Interview/LinkedIn/twosum.js
-----------------------------------------
1 Two Sum 23.0% Easy
-----------------------------------------
var twoSum = function(nums, target) {
      console.log(nums);
      var map = {};

      for(var i=0;i<nums.length;i++){
         if(map[target-nums[i]] !== undefined)
           return [map[target-nums[i]],i];
         else
           map[nums[i]] = i;
      }
    };
------
//python

class Solution(object):
    def twoSum(self, nums, target):
        map = {}
        for i in range(len(nums)):
            if target-nums[i] in map.keys():
                return [map[target-nums[i]],i]
            else:
                map[nums[i]] = i
